pkg/controller/pipeline/pipeline_controller.go

```diff
@@ -239,6 +239,8 @@ func (c *Controller) syncHandler(key string) error {
 					klog.V(8).Info(err, fmt.Sprintf("failed to update pipeline config %s ", key))
 					return err
 				}
+			} else {
+				klog.V(7).Info(fmt.Sprintf("nothing was changed, pipeline '%v'", copyPipeline.Spec))
```

Comment:
The level of the log needs to be changed.

Reply:
Got you.